src/webots/control/WbController.cpp

```diff
@@ -485,6 +485,33 @@ void WbController::setProcessEnvironment() {
     if (mPythonCommand.isEmpty())
       mPythonCommand = WbLanguageTools::pythonCommand(
         mPythonShortVersion, WbPreferences::instance()->value("General/pythonCommand", "python").toString());
+    // read the python shebang (first line starting with #!) to possibly override the python command
+    QString expectedVersion;
```

Comment:
This is only used in the Windows case, couldn't this declaration be moved to line 501?